src/kvstore/raftex/RaftPart.cpp

```diff
@@ -1002,6 +1004,10 @@ typename RaftPart::Role RaftPart::processElectionResponses(
     for (auto& r : results) {
         if (r.second.get_error_code() == cpp2::ErrorCode::SUCCEEDED) {
             ++numSucceeded;
+        } else if (r.second.get_error_code() == cpp2::ErrorCode::E_LOG_STALE) {
+            VLOG(2) << idStr_ << "My last log id is less than " << hosts[r.first]
+                    << ", increase my election interval.";
+            ++weight_;
```

Comment:
We could be more aggressive.  For example,  weight *= 2

Reply:
Make sense.